build.gradle.kts

```diff
@@ -29,6 +29,9 @@ repositories {
 
 // Dependencies are managed with Gradle version catalog - read more: https://docs.gradle.org/current/userguide/platforms.html#sub:version-catalog
 dependencies {
+    implementation(libs.jqa.core.report) {
+        exclude(group = "org.neo4j.community")
```

Comment:
Maybe add a comment why this is excluded e.g. `// jqa core has runtime dependency on neo4j integration tests, the intellij plugin can't handle those integration tests`